x-pack/plugin/src/test/resources/rest-api-spec/test/ml/revert_model_snapshot.yml

```diff
@@ -211,86 +28,6 @@ setup:
         job_id: "revert-model-snapshot"
         snapshot_id: "not_exist"
 
----
```

Comment:
These revert tests were already covered in `org.elasticsearch.xpack.ml.integration.RevertModelSnapshotIT`